tests/integration_tests.rs

```diff
@@ -974,6 +974,30 @@ fn header_full_binary() {
         .stderr("");
 }
 
+#[test]
+fn header_default() {
+    bat()
+        .arg("--paging=never")
+        .arg("--color=never")
+        .arg("--terminal-width=80")
+        .arg("--wrap=never")
+        .arg("--decorations=always")
+        .arg("--style=default")
```

Comment:
Any chance you can add a second test that is a regression test for https://github.com/sharkdp/bat/pull/2119/commits/1b475fb02e92e38ecf4fb53f13bedfa100d7a92e ?

I'm thinking you would simply copy-paste this test but leave out the explicit `.arg("--style=default")`. If you get problems with the default not being right when running tests you don't need to bother with adding a test for that. But if it is an easy test to add, it would be nice to have.